docs/release.rst

```diff
@@ -14,12 +14,22 @@ Unreleased
     # .. warning::
     #    Pre-release! Use :command:`pip install --pre zarr` to evaluate this release.
 
+
+Major changes
+~~~~~~~~~~~~~
+
 * Improve Zarr V3 support, adding partial store read/write and storage transformers.
-  Add two features of the [v3 spec](https://zarr-specs.readthedocs.io/en/latest/core/v3.0.html):
+  `v3 spec <https://zarr-specs.readthedocs.io/en/latest/core/v3.0.html>`_.
   * storage transformers
-  * `get_partial_values` and `set_partial_values`
+  * ``get_partial_values`` and ``set_partial_values``
```

Comment:
Don't think this list will work, but not sure how to fix it. Naive attempts failed. Maybe we ditch the bullet points and do comma separated?

Reply:
The list parsing is fixed in @DON-BRAN's theme update so don't knock yourself out there.